packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -450,6 +437,66 @@ export default class VTable {
         }
         return result;
     }
+
+    private setEmptyTableCells() {
+        for (let i = 0, row; (row = this.table.rows[i]); i++) {
+            for (let j = 0, cell; (cell = row.cells[j]); j++) {
+                if (cell) {
+                    if (!cell.innerHTML || !cell.innerHTML.trim()) {
+                        cell.appendChild(document.createElement('br'));
+                    }
+                }
+            }
+        }
+    }
+
+    public setTableCells() {

```

Comment:
What does this function do? Add comments.

All public functions (especially those will public to customers) need comments.

Reply:
Same here, use a better name. e.g. normalizeTableCellSize.

And I still think this function should be private. When you need to call it from outside, consider do the full normalization of the table instead.

Reply:
fixed the name, the full normalization (normalizeSize) will make the table width/height fixed whereas we don't want fixed width in this scenario, so using only setTableCells instead;